CODEOWNERS

```diff
@@ -12,6 +12,6 @@
 #  https://help.github.com/en/articles/about-code-owners
 #
 
-* @jmacd @MrAlias @Aneurysm9 @evantorrie @XSAM @dashpole @MadVikingGod @pellared @hanyuancheung @dmathieu
+* @MrAlias @Aneurysm9 @evantorrie @XSAM @dashpole @MadVikingGod @pellared @hanyuancheung @dmathieu
```

Comment:
Have you intentionally made changes in this file ? OR this file changed automatically using `make precommit` command ?

Reply:
Oh, I guess it's showing these changes because main branch is updated in upstream. Let me merge main into my current branch. Thanks for pointing out.